rest-api/etl/setup_database_for_etl.sh

```diff
@@ -6,29 +6,40 @@
 # If the databases already exist, drop them prior to running this.
 # Note: this takes a while. Go get some coffee while it's running!
 
-USAGE="tools/setup_database_for_etl.sh --project <PROJECT> --account <ACCOUNT>"
```

Comment:
All tools assume they're being ran from rest-api/ 
For instance, this one sources `tools/auth_setup.sh` on L43 which will fail.
I'm all for the shortest distance being a straight line, but not sure we should change established patterns, this one would require quite a bit of work to establish across the board. I recommend staying with the pattern on this.

Reply:
No problem.  I've updated the usages to show the correct 'etl' directory.